modules/mod_tags_similar/src/Helper/TagsSimilarHelper.php

```diff
@@ -194,7 +194,12 @@ public static function getList(&$params)
 			$query->order($query->rand());
 		}
 
-		$query->setLimit((int) $params->get('maximum', 5));
+		$maximum = (int) $params->get('maximum', 5)
+		if ($maximum > 0)
```

Comment:
```suggestion

		if ($maximum > 0)
```
Empty line before "if" to fix code style error.